strapiExperiment/app/extensions/users-permissions/config/jwt.json

```diff
@@ -0,0 +1,3 @@
+{
+  "jwtSecret": "5afde127-26d8-46a9-be01-95a1ac9a8c71"
```

Comment:
seems peculiar. This is obviously not sensitive. We will have to look into the strapi setup docs to see what this is about.